demos/cuad-query-dep-demo.py

```diff
@@ -0,0 +1,665 @@
+import argparse
```

Comment:
what is this file for?

Reply:
This will ultimately hold a second version of the CUAD demo which will have filters based on supplemental annotations; it's not completed yet, hence the copy.